taca/analysis/analysis.py

```diff
@@ -189,15 +189,7 @@ def _process(run, force_trasfer):
             # In the case of of HiSeq this function computes undetermined indexes for NoIndex lanes
             if not run.compute_undetermined():
                 return
-            # Otherwise I can proceed to QC
-            # Check the run QC
-            run_QC_status = run.check_QC()
-            if run_QC_status is not None:
-                # Store QC results in appropriate file and mail user if failed
-                qc_file = os.path.join(CONFIG['analysis']['status_dir'], 'qc.tsv')
-                # This method is implemented in Runs
-                run.post_qc(qc_file, run_QC_status, log_file=CONFIG['log']['file'],
```

Comment:
This function is the one that sends the first email... the one containing the QC result  that now we do not longer want.... the fact that a function called post_qc sends an email is not an example of code clarity :-D

Anyhow I like to receive an email when transfer starts and when transfer ends also because when transfer starts the stats are already visible on genomics status. So most likely `transfer_run` is the best candidate to be the function sending the email... I can take care of it tomorrow